.github/workflows/test_nix_flake.yml

```diff
@@ -0,0 +1,41 @@
+name: "Test NixOS flake"
+
+on:
+  pull_request:
+  push:
+
+
+jobs:
+  tests:
+    name: Run NixOS Flake and upload generated PDFs
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+  
+    - name: Set up NixOS
+      uses: cachix/install-nix-action@v20
+      with:
+        github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Run nix build
+      run: nix build
+
+    - run: nix flake show
+
+    - run: nix flake metadata
+
+#    - name: Run nix flake check
+#      run: nix flake check
+
+    - run: pwd
+
+    - run: find .
+
+    - run: find ..
+
+    - name: Archive build artifacts
```

Comment:
@Soispha this is where I am trying to find the PDF file, this action will bundle it into a zip file and make it downloadable

Example of a successful run here: https://github.com/matsavage/DND-5e-LaTeX-Character-Sheet-Template/actions/runs/4898220839

Reply:
The file should be symlinked to `./result/sheet.pdf`, if I am not mistaken. I hope, that `find .` is not showing the file because the `result` directory is a symlink, otherwise I'd argue that something with the `nix build` command went wrong.

Reply:
Got it :+1: